examples/crowdfunding/scripts/transfer/common.js

```diff
@@ -0,0 +1,9 @@
+const { executeTransaction } = require('algob');
+
+exports.executeTransaction = async function (deployer, txnParams) {
+  try {
+    await executeTransaction(deployer, txnParams);
+  } catch (e) {
+    console.error('Transaction Failed', e.response ? e.response.error : e.error);
+  }
```

Comment:
do we want to continue execution after the transaction failed? Maybe we should return  true / false?

Reply:
yes, i don't think we need common.js in this example. removing (because the error is correctly displayed without using e.response)

![Screenshot from 2021-01-06 12-06-34](https://user-images.githubusercontent.com/26249903/103737251-d6042d80-5017-11eb-9200-74fb5f8c00f0.png)